src/test/java/com/segment/analytics/android/integrations/adjust/AdjustIntegrationTest.java

```diff
@@ -160,6 +160,55 @@
     verify(adjustInstance).trackEvent(event);
   }
 
+  @Test public void trackWithAnonymousId() throws Exception {
+    AdjustEvent event = mock(AdjustEvent.class);
+    PowerMockito.whenNew(AdjustEvent.class).withArguments("bar").thenReturn(event);
+
+    Traits traits = createTraits() //
+        .putValue("anonymousId", "1234");
+
+    integration.identify(new IdentifyPayloadBuilder().traits(traits).build());
```

Comment:
I don't think these tests are testing what you are meaning to test. How do you know `addSessionPartnerParameter` is being called from the `track` method? `identify` also calls this - so if I were to remove the `addSessionPartnerParameter` from `track`, these tests would still pass.

Reply:
good point